mmcls/datasets/stanford_car.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os.path as osp
+from typing import Optional
+
+import numpy as np
+import scipy.io as sio
+
+from .base_dataset import BaseDataset
+from .builder import DATASETS
+
+
+@DATASETS.register_module()
+class StanfordCar(BaseDataset):
+    """The Stanford Car Dataset. Support the `Stanford Car.
```

Comment:
There are some errors with the format

https://github.com/open-mmlab/mmclassification/blob/d2e505415040bf5329ab218bb6fe3d899f176cd5/mmcls/datasets/mnist.py#L18